### PR TITLE
docs(integration): add information about sessionId duration

### PR DESCRIPTION
**Pull request by herrardo**
SessionId duration information

# Pull request template
 
Inform the integration developers and customers about sessionId duration and way of working.

## Motivation and context
Questions from Customer Success department

- [ ] Dependencies. If any, specify:
- [ ] Open issue. If applicable, link:

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that causes existing functionality to not work as expected)
- [x] Change requires a documentation update

## What is the destination branch of this PR?

- [x] `Main`
- [ ] Other. Specify:

## How has this been tested?

Tests performed according to [testing guidelines](./contributing/tests.md): 

## Checklist:

- [ ] My code follows the **[style guidelines](./CONTRIBUTING.md#style-guides)** of this project.
- [ ] I have performed a **self-review** on my own code.
- [ ] I have **commented** my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the **[documentation](./CONTRIBUTING.md#documentation-style-guide)**.
- [ ] My changes generate **no new warnings**.
- [ ] I have added **tests** that prove my fix is effective or that my feature works.
- [ ] New and existing **unit tests pass locally** with my changes.

### CODE REVIEW

**Comment by support-empathy** [comment]
This is a link to Empathy Platform Docs Portal preview in a development environment. 
 ⚠️ Only for Empathy members. We’re sorry for the inconvenience!
 [Check out EPDocs preview](https://docs-dev.empathy.co/preview/x-1139/index.html)

File: packages/x-components/static-docs/build-search-ui/web-archetype-integration-guide.md [hunk 1]
```diff
@@ -251,6 +251,10 @@ If page reload is not triggered after accepting cookies, update the `consent` pa
 Although cookie acceptance is bound to the generation of the `sessionID` in local storage, Empathy
 does **not use any cookies** in its libraries.
 
+<br/>
+
+The `sessionId` lasts until there are 30 minutes of inactivity. However, every time the shopper interacts with the commerce search (e.g. queries, clicks, etc.), the session expiration time is reset by adding 30 minutes from the time of that interaction. After 30 minutes of shopper inactivity, a new `sessionId` is generated.
```
Comment (mlcorton) [inline packages/x-components/static-docs/build-search-ui/web-archetype-integration-guide.md:256]: Is it `sessionId` or `sessionID`? See that throughout the whole page, both spellings are found!

**Review by mlcorton — COMMENTED** [review]
@herrardo Left a comment about the consistency of the parameter name. Is it `sessionId` or `sessionID`. We need to be consistent.

**Comment by herrardo** [comment]
> @herrardo Left a comment about the consistency of the parameter name. Is `sessionId` or `sessionID`? We need to be consistent.

Fixed!!

**Comment by support-empathy** [comment]
This is a link to Empathy Platform Docs Portal preview in a development environment. 
 ⚠️ Only for Empathy members. We’re sorry for the inconvenience!
 [Check out EPDocs preview](https://docs-dev.empathy.co/preview/x-1139/index.html)

**Review by mlcorton — APPROVED** [review]

**Review by CachedaCodes — APPROVED** [review]

**Review by tajespasarela — APPROVED** [review]

**Pull request merged**